modules/microshift-install-rpms-gitops.adoc

```diff
@@ -0,0 +1,55 @@
+// Module included in the following assemblies:
+//
+// microshift/microshift-install-rpm.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-installing-rpms-for-gitops_{context}"]
+== Installing the GitOps ArgoCD manifests from an RPM package
+
+You can use a lightweight version of the OpenShift GitOps with {microshift-short} to help manage your applications. Install the necessary ArgoCD manifests using an RPM package. This RPM package included the necessary manifests that runs core Argo CD. 
+
+.Prerequisites
+
+* You installed {microshift-short} version 4.14 or higher
+* Additional RAM storage of 250MB recommended 
+
+.Procedure
+
+. Enable the GitOps repository with the subscription manager by running the following command:
++
+[source,terminal]
+----
+$ sudo subscription-manager repos --enable=gitops-1.12-for-rhel-9-<architecture>-rpms
+----
+
+. Install the GitOps packages by running the following command: 
```

Comment:
```suggestion
. Install the GitOps package by running the following command: 
```
In this case, only 1 package is installed, `microshift-gitops-1.12.0-36.el9.x86_64.rpm`. But `dnf install` command may install multiple packages.